test/e2e/infinispan/encryption_test.go

```diff
@@ -45,6 +46,31 @@ func TestTLSWithExistingKeystore(t *testing.T) {
 	checkRestConnection(client_)
 }
 
+func TestTLSConditionWithBadKeystore(t *testing.T) {
+	t.Parallel()
+	defer testKube.CleanNamespaceAndLogOnPanic(t, tutils.Namespace)
+
+	// Create a resource without passing any config
+	replicas := 2
+	spec := tutils.DefaultSpec(t, testKube, func(i *ispnv1.Infinispan) {
+		i.Spec.Replicas = int32(replicas)
```

Comment:
There's no need to explicitly define this as the test will work with the default value of 1.